README.md

```diff
@@ -69,6 +69,88 @@ We're now on the Maven Central Repository. Add the following to your POM file:
 
 The driver can be downloaded from the [Microsoft Download Center](https://www.microsoft.com/en-us/download/details.aspx?displaylang=en&id=11774)
 
+##Dependencies
+This project has following dependencies: 
+
+Compile Time:
+
+ - `azure-keyvault`
+
+Test Time:
+
+ - `junit:jar`
+
+If anybody wants to use driver as a run-time dependency but not using Azure Web Services for authentication then you either describe as optional dependency or exclude azure.
+
+```
+<dependency>
+	<groupId>com.microsoft.sqlserver</groupId>
+	<artifactId>mssql-jdbc</artifactId>
+	<version>6.1.0.jre8</version>
+	<scope>runtime</scope>
+	<exclusions>
+		<exclusion>
+		         <groupId>com.microsoft.azure</groupId>
+		         <artifactId>azure-keyvault</artifactId>
+		</exclusion>
+    </exclusions>
+</dependency>
+```
+
+###Dependency Tree
```

Comment:
Instead of listing the output for the dependency tree, perhaps you can describe how someone can get this information for themselves?  (This list will be hard to maintain if it changes in the future.)